inference-engine/thirdparty/clDNN/api/cldnn/graph/program.hpp

```diff
@@ -190,7 +190,7 @@ struct program {
     bool extract_and_remove(program_node& node);
 
     // Fuses two nodes into fused_node and removes peer_node from graph
-    void fuse_nodes(program_node& fused_node, program_node& peer_node, std::map<primitive_id, std::vector<primitive_id>>* fusing_history);
+    void fuse_nodes(program_node& fused_node, program_node& peer_node, std::map<primitive_id, std::map<primitive_id, size_t>>* fusing_history);
```

Comment:
I think vector of pair will be enough for this purpose. map is much heavior and we dont need heavy  access using key. We'll just iterate on it. also vector is contiguous so that better cache efficiency.

Reply:
Fixed it. thanks.